spec/pharos/phases/label_node_spec.rb

```diff
@@ -102,7 +102,7 @@
             labels: { },
           },
           spec: {
-            taints: [ { key: 'node-role.kubernetes.io/master', value: nil, effect: 'NoSchedule' } ],
```

Comment:
Needs merge for #471 changes, specs are failing after master merge... I wonder why this doens't show up in drone.